chain/network/src/stats/rate_counter.rs

```diff
@@ -59,9 +59,7 @@ impl RateCounter {
 
 /// Returns timestamp in milliseconds.
 fn millis_since_epoch() -> u128 {
-    let since_epoch = SystemTime::now()
-        .duration_since(SystemTime::UNIX_EPOCH)
-        .unwrap_or_else(|_| Duration::new(0, 0));
+    let since_epoch = SystemTime::now().duration_since(SystemTime::UNIX_EPOCH).unwrap_or_default();
```

Comment:
you're removing this method in a different PR anyway (so you can ignore it here)

Reply:
Ok, I can fix merge requests later.